### ISSUE TITLE
WL Leonine Frenzy SHOULD crit as it did on live.

### ISSUE THREAD

**Issue by redman012** [WarEmu/WarBugs#17540]
**Expected behavior and actual behavior:**
WL has a 2 second internal cooldown on LF as it did on live, but move will not crit as it should.

**Steps to reproduce the problem:**

Go to test dummy and use LF move cannot crit.

**Testing Screenshots/Videos/Evidences (always needed):**

**Comment by Zomega-WAR** [comment]
Intentional change from 2017: https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=23834

**Comment by redman012** [comment]
> Intentional change from 2017: https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=23834
> 
> "- Reworked Leonine Frenzy - It can no longer crit on the damage procs but will now add strikethrough, armour penetration and a speed increase to the pet. Visual effect also added."

Those changes were removed after WL's got put back to normal state after the Torq changes. WL pet does NOT get armor pen or speed increase.  I just tested it. Pet will attack for 223 without LF and 223 with LF. Also does not have any type of speed with LF. No buffs say other wise. 

So, can we please fix LF as it is not working as you listed.

**Comment by lilimx** [comment]
Hey,

any source that these "Torq" changes were removed after he left the team.

**Comment by redman012** [comment]
> Hey,
> 
> any source that these "Torq" changes were removed after he left the team.

Yes, they were removed. Team is not interested in fixing them or restoring.

**Comment by lilimx** [comment]
> > Hey,
> > any source that these "Torq" changes were removed after he left the team.
> 
> Yes, they were removed. Team is not interested in fixing them or restoring.

Hey, 

source that the "Team" said that is not interested in fixing it or restoring?
